README.md

```diff
@@ -221,9 +221,37 @@ $ tree -L 1 .
 └── go
 ```
 
-To regenerate the test cases, navigate into the **go** directory and run
-`GO111MODULE=off go run exercises/practice/<exercise>/.meta/gen.go`. You should see that the
-`<exercise>/cases_test.go` file has changed. Commit the change.
+To keep track of which tests are implemented by the exercise the file `.meta/tests.toml` is used by [configlet](https://github.com/exercism/configlet).
+
+A `tests.toml` file has this format:
```

Comment:
We should not include the full format of `tests.toml` here. Just mentioning the file is fine.